apps/watchdog/src/deploy_with_compose.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+#![forbid(unsafe_code)]
+
+use std::path::PathBuf;
+use std::process::{Command, Output};
+
+use slog::{info, Logger};
+use tokio::time::{sleep, Duration};
+
+use crate::image::{Explorer, OcamlDebugger, Sandbox, TezedgeDebugger, WatchdogContainer};
+use crate::node::{OcamlNode, TezedgeNode};
+
+// TODO: use external docker-compose for now, should we manage the images/containers directly?
```

Comment:
@adonagy 
what about this TODO? please create jira for that or remove comment, if not needed